src/Payloads/PaymentPayload.php

```diff
@@ -103,6 +103,23 @@ protected static function delivery(WC_Order $order)
         ];
     }
 
+    /**
+     * Get the total discount of the order.
+     *
+     * @param  \WC_Order  $order
+     * @return float
+     */
+    protected static function discount(WC_Order $order)
+    {
+        return array_reduce($order->get_items(['line_item', 'fee']), function ($total, $item) {
+            if ($item->get_total() < 0) {
+                $total += abs($item->get_total() + $item->get_total_tax());
```

Comment:
@Tofandel I changed the item mapping here, because theoritically it is possible to have bigger quantities of one fee, line item than one. Even if it's not very common. Can you confirm, that this works for you, please?

Reply:
I tested it, and yes it works fine 👍

Reply:
Great, thanks! Yep, soon I'll release it.